cvcl/models.py

```diff
@@ -3,25 +3,25 @@
 
 class User(models.Model)
     username = models.CharField(max_length=30)
-    password = models.CharField(max_length=30)
+    usergroup = models.ManyToManyField(UserGroup)
```

Comment:
for consistency you use make it `user_group` - just like you did `start_date`, because it's two words